app/src/main/res/layout/item_saved_file.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:minHeight="60dp"
+        app:cardBackgroundColor="#FFFBEC"
```

Comment:
lets use colour from the palette for more consistancy

Reply:
Can you please help me with this. I am not able to refer with @attr/ in this file

Reply:
And when i referred from ?attrib, it is causing runtime crash